<?xml version="1.0" encoding="UTF-8"?>

<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="fixed_camera">
  <xacro:include filename="$(find gazebo_description)/urdf/depth.xacro" />

  <link name="world" />
  <link name="fixed_camera_link">
    <visual>
      <geometry>
	<box size="0.01 0.01 0.01" />
	<origin xyz="0 0 -0.5" rpy="0 0 0" />
      </geometry>
    </visual>
    <inertial>
      <mass value="1" />
      <inertia ixx="1" iyy="1" izz="1" ixy="0.0" ixz="0.0" iyz="0.0" />
    </inertial>
  </link>
  <xacro:camera_gazebo_sensor xyz="1 1 0" rpy="0.0 0.0 0.0"
  			      parent="fixed_camera_link"
  			      sensor_name="fixed_camera" />

  <joint name="fixed_joint" type="fixed">
    <parent link="world" />
    <child link="fixed_camera_link" />
    <origin xyz="0 0 1.0" rpy="0 0 0" />
  </joint>
</robot>
